src/struct.cr

```diff
@@ -73,12 +73,12 @@ struct Struct
   # Returns a hash value based on this struct's instance variables hash values.
   #
   # See also: `Object#hash`
```

Comment:
The old documentation of `hash` is detached from anything, you should remove it if you don't need it.